db-engine/src/test/java/com/splicemachine/db/iapi/types/SQLCharTest.java

```diff
@@ -164,8 +166,11 @@ public void testColumnStatistics() throws Exception {
                 Assert.assertEquals(1000,stats.selectivity(new SQLChar()));
                 Assert.assertEquals(347,stats.selectivity(new SQLChar(new char[]{'A'})));
                 Assert.assertEquals(347,stats.selectivity(new SQLChar(new char[]{'F'})));
-                Assert.assertEquals(1404.0d,(double) stats.rangeSelectivity(new SQLChar(new char[]{'C'}),new SQLChar(new char[]{'G'}),true,false),RANGE_SELECTIVITY_ERRROR_BOUNDS);
-                Assert.assertEquals(700.0d,(double) stats.rangeSelectivity(new SQLChar(),new SQLChar(new char[]{'C'}),true,false),RANGE_SELECTIVITY_ERRROR_BOUNDS);
+                double range = stats.rangeSelectivity(new SQLChar(new char[]{'C'}),new SQLChar(new char[]{'G'}),true,false);
+                Assert.assertTrue(range + " did not match expected", (range == 1372.0d || range == 1404.0d));
```

Comment:
Is a direct comparison of double data type appropriate? Should we give it an error bound like before?